### PR TITLE
Added a Tile

### PR DESCRIPTION
**Pull request by naveentherocker**
Reference #19

### CODE REVIEW

**Comment by niccokunzmann** [comment]
Thanks for your pull request.. let's walk through what needs to be done to add a tile to the current labyrinth. 
What would you do next when you read this https://github.com/fossasia/labyrinth/issues/19#issuecomment-353604274 ?

**Comment by niccokunzmann** [comment]
The end of the tiles.js file could look like this:

```
  black: Object.assign({}, OpenDoors, {
    createImages: function() {
      this.wallTop = this.createImage("tiles/rooms/door/top.svg");
      this.wallRight = this.createImage("tiles/rooms/door/right.svg");
      this.ground = this.createImage("tiles/rooms/floor/black.svg");
    },
  })
};
```

if you want people to enter from all sides.

**Comment by naveentherocker** [comment]
Configured the Tile.js

File: js/tiles.js [hunk 1]
```diff
@@ -177,3 +177,11 @@ const door = {
     },
   })
```
Comment (niccokunzmann) [inline js/tiles.js]: please delete this and the following line, so you are inside the door variable.

Comment (niccokunzmann) [reply js/tiles.js]: or maybe I was wrong... The idea is to have your tile available as "door.black" in JavaScript...

Comment (naveentherocker) [reply js/tiles.js]: Sure!

Comment (niccokunzmann) [reply js/tiles.js]: When you did this, you can edit lavels.js to include the tile somewhere... Then you can navigate there or start there.

**Comment by naveentherocker** [comment]
Now it comes inside the door var. It's fine i guess.

**Comment by niccokunzmann** [comment]
Now, see this line: 

https://github.com/fossasia/labyrinth/blob/master/js/levels.js#L16

if you replace `PlayerStartsAt(door.both)` with `PlayerStartsAt(door.black)`, you should start at your black tile.

**Comment by naveentherocker** [comment]
Yeah.

**Comment by niccokunzmann** [comment]
It works :) ?

**Comment by niccokunzmann** [comment]
Thanks!

**Comment by niccokunzmann** [comment]
If you would like to get this approved as a GCI task for adding a tile, you can modify the tile a bit, e.g. add the text fossasia on the ground of the tile.

**Comment by naveentherocker** [comment]
Sure and I will come with few more tiles by today.

**Comment by niccokunzmann** [comment]
If you do that, you can also think about a new theme... You can extent the door theme or create a new one like rivers, bridges, pipes, ... whatever you like.

**Comment by naveentherocker** [comment]
Sure!

**Pull request merged**